js/utils/resyncBlockchain.js

```diff
@@ -0,0 +1,140 @@
+import $ from 'jquery';
+import {
+  getServer,
+  events as serverConnectEvents,
+} from './serverConnect';
+import { openSimpleMessage } from '../views/modals/SimpleMessage';
+import { Events } from 'backbone';
+import app from '../app';
+
+const events = {
+  ...Events,
+};
+
+export { events };
+
+let server = getServer();
+
+// If you change this, be sure to change anywhere in the GUI you may have output how
+// long its unavailable (e.g. Advanced Settings).
+const resyncInactiveTime = 1000 * 60 * 60 * 1; // 1 hour
```

Comment:
You could muck with this if you want to test a resync being unavailable for a certain amount of time and not having to wait the hour that's coded here.